### PR TITLE
Update pgcli to 2.1.1

### PR DESCRIPTION
**Pull request by g5pw**
#### Description

Upgrade `pgcli` to 2.1.1, needs `sqlparse` 0.3.3

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.6 18G103
Xcode 11.1 11A1027

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-sqlparse/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-sqlparse
-version             0.2.4
+version             0.3.0
```
Comment (neverpanic) [inline python/py-sqlparse/Portfile]: In the commit message, did you mean: `python/py-sqlparse`?

Comment (g5pw) [reply python/py-sqlparse/Portfile]: Uh, yes, yes I did.

Comment (neverpanic) [reply python/py-sqlparse/Portfile]: Did you want to fix that, or do you want to merge as-is?

Comment (g5pw) [reply python/py-sqlparse/Portfile]: Yes, I'll fix it later tonight!

Comment (g5pw) [reply python/py-sqlparse/Portfile]: ready to merge.

**Pull request merged**